pages.id/common/cp.md

```diff
@@ -0,0 +1,27 @@
+# cp
+
+> Membuat salinan file dan direktori
+
+- Membuat salinan file ke lokasi lain:
+
+`cp {{jalan/menuju/file_sumber.ext}} {{jalan/menuju/file_tujuan.ext}}`
+
+- Menyalin file ke direktori lain, dengan nama yang sama:
+
+`cp {{jalan/menuju/file_sumber.ext}} {{jalan/menuju/direktori_tujuan}}`
+
+- Menyalin sebuah direktori secara beserta isinya ke lokasi lain (jika tujuan sudah ada, direktori tersebut disalin ke dalamnya):
+
+`cp -R {{jalan/menuju/source_directory}} {{jalan/menuju/direktori_tujuan}}`
```

Comment:
Why `source_directory` is not translated? I don't know the language, but I think this is the translation:
```suggestion
`cp -R {{jalan/menuju/direktori_sumber}} {{jalan/menuju/direktori_tujuan}}`
```

Reply:
fixed per suggestions